README.md

```diff
@@ -67,3 +67,6 @@ Awesome List for Phoenix.  Compiles a list of awesome references from the az-web
 * RetailMeNot
 * Shutterfly
 * Tuft & Needle
+
+## Meetups
```

Comment:
Can you move this above the list of companies? That list could be huge, so it would be nice if it was last.